include/psa/crypto.h

```diff
@@ -932,7 +932,7 @@ psa_status_t psa_hash_compare(psa_algorithm_t alg,
                               const uint8_t *input,
                               size_t input_length,
                               const uint8_t *hash,
-                              const size_t hash_length);
```

Comment:
Does this mean that we need to update the API reference? This is still const in RC2...

Reply:
Indeed, thanks for noticing that.

Reply:
And I also fixed this in the similar function for MAC, because it's easier to fix it now than track it for later.

Reply:
I'll update the spec - this doesn't actually change the interface, but it is not standard practice to use this in the API declaration.

Reply:
@athoelke  https://github.com/ARMmbed/psa-crypto-api/pull/366

Reply:
I saw that - and I've patched it with the corrected header file.